.azure/pipelines/jobs/default-build.yml

```diff
@@ -294,12 +295,12 @@ jobs:
         condition: failed()
         displayName: Upload cores
 
-    - ${{ if and(eq(parameters.isTestingJob, true), ne(parameters.jobName, 'Windows_Templates_Test')) }}:
+    - ${{ if and(eq(parameters.isTestingJob, true), ne(parameters.enablePublishTestResults, false)) }}:
```

Comment:
This aligns where we execute this step (`Publish js test results`) with where / when Arcade executes its `Publish XUnit Test Results` step.